### PR TITLE
add new mysql 8 keywords

### PR DESCRIPTION
**Pull request by mojoaxel**
mysql v8 added some new keywords.
The full keyword list can be found here: https://dev.mysql.com/doc/mysqld-version-reference/en/keywords-8-0.html

- adds new v8 keyword and removs keywords that are no longer "active".
- removed one duplicated keyword `CURRENT`

### CODE REVIEW

**Review by gjsjohnmurray — APPROVED** [review]
Thanks for submitting this.

**Comment by gjsjohnmurray** [comment]
@mojoaxel please update this branch from its base branch, or set the checkbox to allow maintainers such as me to do this for you. Until that happens I'm not able to merge it.

**Comment by mojoaxel** [comment]
🎉

**Pull request merged**
